models/repo/mirror.go

```diff
@@ -113,6 +113,17 @@ func UpdateMirror(m *Mirror) error {
 	return updateMirror(db.GetEngine(db.DefaultContext), m)
 }
 
+func touchMirror(e db.Engine, m *Mirror) error {
+	m.UpdatedUnix = timeutil.TimeStampNow()
+	_, err := e.ID(m.ID).Cols("updated_unix").Update(m)
+	return err
+}
+
+// TouchMirror updates the mirror updatedUnix
+func TouchMirror(m *Mirror) error {
+	return touchMirror(db.GetEngine(db.DefaultContext), m)
+}
```

Comment:
Since we are going to use `ctx` everywhere, so I think a `TouchMirror(ctx context.Context, m *Mirror) error` is enough.

Reply:
The main reason I didn't do that is that I didn't want to refactor the rest of file, but I guess we can do that.